src/pe_source/data/sixgill/config.py

```diff
@@ -0,0 +1,46 @@
+"""Get cybersixgill API token."""
+
+# Standard Python Libraries
+from configparser import ConfigParser
+import os
+
+# Third-Party Libraries
+from importlib_resources import files
+import requests
+
+# Configuration
+SECTION = "sixgill"
+REPORT_DB_CONFIG = files("pe_reports").joinpath("data/database.ini")
+
+
+def token():
+    """Retrieve bearer token from Cybersixgill client."""
+    if os.path.isfile(REPORT_DB_CONFIG):
+        parser = ConfigParser()
+        parser.read(REPORT_DB_CONFIG, encoding="utf-8")
+        if parser.has_section(SECTION):
+            params = parser.items(SECTION)
+            _id, _secret = params[0], params[1]
+            client_id = _id[1]
+            client_secret = _secret[1]
+        else:
+            raise Exception(
+                "Section {} not found in the {} file".format(SECTION, REPORT_DB_CONFIG)
+            )
+    else:
+        raise Exception(
+            "Database.ini file not found at this path: {}".format(REPORT_DB_CONFIG)
+        )
+    url = "https://api.cybersixgill.com/auth/token/"
+    headers = {
+        "Content-Type": "application/x-www-form-urlencoded",
+        "Cache-Control": "no-cache",
+    }
+    payload = {
+        "grant_type": "client_credentials",
+        "client_id": client_id,
+        "client_secret": client_secret,
+    }
+    resp = requests.post(url, headers=headers, data=payload).json()
+    token = resp["access_token"]
+    return token
```

Comment:
You can similary simplify this - you should look for other places where this pattern can be done:
```suggestion
    return resp["access_token"]
```

Reply:
I assume I should keep this as is right? Only the simple ones?
<img width="683" alt="Screen Shot 2022-03-08 at 10 52 18 AM" src="https://user-images.githubusercontent.com/79927030/157274295-26a42d70-66d8-4f8e-80e5-0de9f3152139.png">

Reply:
Yeah, that's your judgement call to make.  I'm fine if you want to keep the more complicated ones like this, but definitely change the simple ones.

Reply:
Check out this commit: https://github.com/cisagov/pe-reports/pull/117/commits/0a1df35f77ba87457822eb82f3a009fa201f284e

Reply:
Yup, that all looks great 👍